src/partisan_util.erl

```diff
@@ -269,3 +270,29 @@ term_to_iolist_(T) ->
     %% fallback clause
     <<131, Rest/binary>> = term_to_binary(T),
     Rest.
+
+iolist_to_term([131, T]) ->
```

Comment:
why the magic numbers?

Reply:
term_to_iolist(Term) ->
     [131, term_to_iolist_(Term)].